source/scone/output/helpers/arguments_to_cells_converter.d

```diff
@@ -87,7 +87,7 @@ class ArgumentsToCellsConverter(Args...)
             {
                 continue;
             }
-            else static if (is(typeof(arg) : Color))
+            else static if (is(typeof(arg) == Color))
```

Comment:
Right now `Color` is just silently ignored. Shouldn't this simply be converted to a string?

Reply:
That isn't really related to this PR.

It can be discussed sometime else.